tests/functional_test.js

```diff
@@ -1,41 +1,41 @@
 'use strict';
 
-var path   = require('path');
+var path = require('path');
 var assert = require('assert');
-var walk   = require('fs-walk');
-var async  = require('async');
+var walk = require('fs-walk');
+var async = require('async');
 
-var digest  = require(path.join(__dirname, '..', 'lib', 'helpers')).sri.digest;
+var digest = require(path.join(__dirname, '..', 'lib', 'helpers')).sri.digest;
 var helpers = require(path.join(__dirname, 'test_helper'));
-var config  = helpers.config();
+var config = helpers.config();
 
 var expectedHeaders = {
-    date: undefined,
-    etag: undefined,
-    expires: undefined,
+    'date': undefined,
```

Comment:
I see, so if some are quoted, it recommends that all in the object are for consistency. That makes sense. I have no strong feelings on it, was just wondering.